app/controllers/graphql_devise/concerns/set_user_by_token.rb

```diff
@@ -7,6 +7,33 @@ module Concerns
     SetUserByToken.module_eval do
       attr_accessor :client_id, :token, :resource
 
+      class_methods do
+        def set_resource_by_model(models, **kwargs)
```

Comment:
Should we have a sensible default for `models`. Just thinking about the (probably) most common case, where there is only one auth model, it may feel redundant to pass is explicitly.

Reply:
As in assume `User`?